openedx/features/pakx/lms/overrides/signals/handlers.py

```diff
@@ -0,0 +1,25 @@
+"""
+Progress Email related signal handlers.
+"""
+
+from logging import getLogger
+from django.dispatch import receiver
+
+from student.models import EnrollStatusChange
+from student.signals import ENROLL_STATUS_CHANGE
+from openedx.features.pakx.lms.overrides.tasks import add_enrollment_record, remove_enrollment_record
+
+
+log = getLogger(__name__)
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def copy_active_course_enrollment(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Names of `copy_active_course_enrollment` and `copy_active_course_enrollments` are very similar and bit confusing